samples/ingestion/ingestion-client/Connector/Serializable/Conversations/RedactedTranscriptContent.cs

```diff
@@ -0,0 +1,63 @@
+﻿// <copyright file="RedactedTranscriptContent.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace Connector.Serializable.Language.Conversations
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class RedactedTranscriptContent
+    {
+        /// <summary>
+        /// Gets or sets PII Redacted Itn version of the transcript input.
+        /// </summary>
+        [JsonProperty(PropertyName = "itn")]
+        public string Itn
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// Gets or Sets PII Redacted lexical version of the transcript input.
+        /// </summary>
+        [JsonProperty(PropertyName = "lexical")]
+        public string Lexical
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// Gets or sets PII redacted maskedItn version of the transcript input.
+        /// </summary>
+        [JsonProperty(PropertyName = "maskedItn")]
+        public string MaskedItn
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// Gets or Sets PII redacted display version of the transcript input.
+        /// </summary>
+        [JsonProperty(PropertyName = "text")]
+        public string Text
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// Gets  Redacted audio timing information for speech input.
+        /// </summary>
+        [JsonProperty(PropertyName = "redactedAudioTimings")]
+        public IList<AudioTiming> RedactedAudioTimings
```

Comment:
In the current output Json format, we are losing a lot of detail/fidelity from the language API response because we are just redacting the text directly. Instead of patching the speech response I think we should add the language service response in parallel to the JSON so we don't have loss of information (e.g. customer needed original as well as redacted text, or redaction timings, pii categories etc). 

IMO, we should stick with the documented API responses from the respective services and ADD any helper fields to make consumption easier if/when needed. If the language service responses have some conflicting/confusing names as compared to the speech response then we can look at transforms but let's start with the existing API responses.

Bidisha, it would be great if you can propose an update to the ingestion client output JSON format that will retain all details and works for PII, summary and any other insights we want to add in the future. Happy to help brainstorm as needed.

Reply:
One option is adding the response directly to the SpeechTranscript object. The output will look something like this. @shahidkhuram @HenryvanderVegte feel free to add your comments here:

````
{
  "source": "https://<sourceurl>",
  "timeStamp": "2022-07-21T19:12:04Z",
  "durationInTicks": 116400000,
  "duration": "PT11.64S",
  "combinedRecognizedPhrases": [
    {
      "channel": 0,
      "lexical": "my name is jane doe",
      "itn": "my name is jane doe",
      "maskedITN": "",
      "display": "My name is jane doe."
    }
  ],
  "recognizedPhrases": [
    {
      "recognitionStatus": "Success",
      "channel": 0,
      "speaker": 0,
      "offset": "PT0.13S",
      "duration": "PT10.77S",
      "offsetInTicks": 1300000,
      "durationInTicks": 107700000,
      "nBest": [
        {
          "confidence": 0.8295387,
          "lexical": "my name is jane doe",
          "itn": "my name is jane doe",
          "maskedITN": "",
          "display": "My name is jane doe.",
          "sentiment": {
            "negative": 0,
            "neutral": 0,
            "positive": 0
          },
          "words": []
        }
      ]
    }
  ],
  "conversationAnalyticsResults": {
    "PiiResults": {
      "conversations": [
        {
          "id": "0",
          "conversationItems": [
            {
              "id": "0",
              "redactedContent": {
                "itn": "my name is ********",
                "lexical": "my name is ********",
                "text": "My name is ********."
              },
              "entities": [
                {
                  "text": "jane doe",
                  "category": "Name",
                  "offset": 11,
                  "length": 8,
                  "confidenceScore": 0.81
                }
              ]
            }
          ],
          "warnings": []
        }
      ],
      "errors": []
    },
    "summarizationResults": {
      "conversations": [
        {
          "id": "0",
          "summaries": [],
          "warning": []
        }
      ],
      "errors": []
    }
  }
}

Reply:
I thought removing the Pii completely from the transcript is intended? 
Please check with Panos as he got more customer insights regarding this.

Reply:
what's the id here? 
does it correspond with the order of items in the recognizedPhrases?

Reply:
for conversation item id: we are using the $"{item.Channel}_{item.Offset}
for conversation id we are using a GUID as it just the identifier for the whole conversation and do not want to tie it to any channel.

Reply:
@bidisha-c {item.Channel}_{item.Offset} is a bit confusing. How come channel and offset/id are not unique fields on the response? Alternatively each channel can be a separate list if you want to drop the offset.

@HenryvanderVegte, I believe you said you had a proposal you would share an example for on how to merge this at the segment level? Could you please share? Ideally also share an example on how that would work when language insights apply to multiple segments, or don't have a clear correlation (e.g. summarization)

Reply:
As long as Offset is unique, we do not really need the channel here. I can update to have the Offset only.

Reply:
@bidisha-c can you update the example here to reflect your changes?

I agree with Khuram that {item.Channel}_{item.Offset}  is a bit confusing. We could also just have a channelIndex as part of the json. Even if it is not part of the original conversationAnalyticsResults, it should scale if we just add it everywhere like:

`"conversationAnalyticsResults": {
    "PiiResults": {
      "conversations": [
        {
          "id": "0",
          "conversationItems": [
            {
              "id": "PT0.13S",
              **"channel": "0",**
              "redactedContent": {
                "itn": "my name is ********",
                "lexical": "my name is ********",
                "text": "My name is ********."
              },
              "entities": [
                {
                  "text": "jane doe",
                  "category": "Name",
                  "offset": 11,
                  "length": 8,
                  "confidenceScore": 0.81
                }
              ]
            }
          ],
          "warnings": []
        }
      ],
      "errors": []
    },
    "summarizationResults": {
      "conversations": [
        {
          "id": "0",
          "summaries": [],
          "warning": []
        }
      ],
      "errors": []
    }
  }`

Reply:
or if you want to keep the format, please add some documentation in code and in the readme since the id format is not intuitive imo

Reply:
ID is the field we use in TA for uniquely identifying conversation turns which is missing speech output. I could not find a unique identiofier for each recognized phrase. The customer can disregard it as it we are augmenting the TA output to have the offset and channel to map to the speech response. If this is still confusing we can remove the ID field itself.

Reply:
I have addressed most of the comments as part of this commit. 

1. Support for chunking
2. Support for combinedRedactedContent
3. Replacing the conversation item ID with an index in response to simplify for customer.
Here is a sample output 

```
"conversationAnalyticsResults":` {
    "conversationPiiResults": {
      "combinedRedactedContent": [
        {
          "channel": "1",
          "lexical": "hi how can I help you",
          "itn": "hi how can I help you",
          "display": "Hi how can I help you?"
        },
        {
          "channel": "1",
          "lexical": "my name is ********",
          "itn": "my name is ********",
          "display": "My name is ********."
        }
      ],
      "conversations": [
        {
          "conversationItems": [
            {
              "id": "1",
              "channel": "0",
              "offset": "PT0.15S",
              "redactedContent": {
                "itn": "hi how can I help you",
                "lexical": "hi how can I help you",
                "text": "hi how can I help you?"
              },
              "entities": []
            },
			{
              "id": "2",
              "channel": "1",
              "offset": "PT0.35S",
              "redactedContent": {
                "itn": "my name is ********",
                "lexical": "my name is ********",
                "text": "My name is ********."
              },
              "entities": [
                {
                  "Text": "Jane Doe",
                  "Offset": 11,
                  "Length": 8,
                  "Score": 0.0,
                  "Category": "Name"
                }
              ]
            }
          ],
          "warnings": []
        }
      ]
    }
  }
```

Reply:
Looks good! Thanks!